package.json

```diff
@@ -100,5 +100,5 @@
       "optional": true
     }
   },
-  "packageManager": "yarn@3.2.0"
+  "packageManager": "yarn@3.2.3"
```

Comment:
@xzhavilla I updated yarn to its latest stable version. CI seems to be working now.